### ISSUE TITLE
Window flashes when on fullscreen (Sway)

### ISSUE THREAD

**Issue by palb91** [qutebrowser/qutebrowser#6025]
**Version info**:
- OS: ArchLinux 5.10.5-arch1-1
- WM: SwayWM
- qutebrowser: https://paste.the-compiler.org/view/0d1fa07b

**Does the bug happen if you start with `--temp-basedir`?**:
- Yes

**Description**
- On Swaywm (and probably Wayland in general), when Qutebrowser is in fullscreen, the window flashes when the context menu is open or when a tooltip appears. The window seems to repeatedly leave and re-enter fullscreen state.
- It's more visible when qutebrowser is on the left of the screen and takes less than 50% of the screen.
- It does not append on X11.
- It's probably the same bug than #3838.

**How to reproduce**
1. On SwayWM: `$ qutebrowser -T https://github.com`
2. Enable fullscreen
3. Right-click in the page or put the mouse pointer hover a logo at the very bottom-right of the page (and wait that the tooltip appears)
4. Move the mouse, the window should flash.

**Comment by palb91** [comment]
Falkon has the same problem. So it seems to be QT's fault.
I just tried again on i3, no problem, but with `QP_QPA_PLATFORM=xcb`, the bug is still here.

**Comment by palb91** [comment]
I tried to disable them, using `qt5ct`, I finally found ~a solution~ something.
It seems that it's not possible in qss to set something like `display: none;`, but the following code works:

In qt5ct -> Style Sheet -> <kbd>Create</kbd> -> `tooltip-disabled.qss`:
```css
QToolTip{
  max-width: 0px;
}
```
Even if they are not completely disabled, it is enough to stop flashing.
There is still the problem of the contextual menu, but I don't use it.
It's clearly not a solution as tooltips are very relevant for some websites, but this workaround works.
